### DISCUSSION TITLE
Was the youtube-dlc-2020.10.9 supposed to fix the bandcamp issue?

### DISCUSSION THREAD

**Discussion by badpythoncode** [blackjack4494/youtube-dlc#179] [General]
I thought I read on reddit that the youtube-dlc-2020.10.9 release was supposed to fix the issue with bandcamp albums not being downloaded.

But I just installed it and the problem is still there.

**Comment by blackjack4494** [comment]
@badpythoncode 
Single tracks should work again. Album/Playlist don't as far as I know. But before that bandcamp was completely broken. It's easier to apply fixes step by step.

**Reply by badpythoncode** [reply]
Thanks :)

Great work everyone :D
